superset-frontend/cypress-base/cypress/integration/explore/control.test.js

```diff
@@ -152,16 +148,13 @@ describe('AdhocFilters', () => {
     cy.verifySliceSuccess({ waitAlias: '@postJson' });
 
     cy.get('[data-test=adhoc_filters]').within(() => {
-      cy.get('.Select-control').click({ force: true });
-      cy.get('input').type('name', { force: true });
-      cy.get('.VirtualizedSelectFocusedOption').trigger('mousedown').click();
+      cy.get('.Select__control').click();
+      cy.get('input[type=text]').type('name{enter}');
```

Comment:
The new `react-select` adds a `input:text` as well as `input:hidden` for form values.